src/main/java/com/google/devtools/build/lib/buildtool/BuildRequestOptions.java

```diff
@@ -515,6 +515,15 @@ public ThreadConverter() {
   @Nullable
   public PathFragment aqueryDumpAfterBuildOutputFile;
 
+  @Option(
+    name = "skip_incompatible_explicit_targets",
+    defaultValue = "false",
+    documentationCategory = OptionDocumentationCategory.EXECUTION_STRATEGY,
+    effectTags = {OptionEffectTag.LOADING_AND_ANALYSIS},
+    help = "Skip incompatible targets that are explicitly listed on the command line."
```

Comment:
Let's add a short clarifier that the alternative is an error? It'd also be nice to have a "See: <url>" for the Bazel URL that documents incompatible targets.

Reply:
Good idea! Let me know if there is a more appropriate URL to include.